src/evrythng/tests/test_entities.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import os
+from evrythng.entities import actions
+from evrythng.entities import products
```

Comment:
Check out the imports section in PEP 8: https://www.python.org/dev/peps/pep-0008/#imports

(The only thing missing here is a blank line between stdlib imports and local imports)